apisix/plugins/hmac-auth.lua

```diff
@@ -29,6 +28,7 @@ local core       = require("apisix.core")
 local hmac       = require("resty.hmac")
 local consumer   = require("apisix.consumer")
 local ngx_decode_base64 = ngx.decode_base64
+local try_read_attr = core.table.try_read_attr
```

Comment:
we do not need to cache it, call `core.table.try_read_attr` is simpler.

Reply:
please fix the similar points

Reply:
OK. Let me fix those points.